meta-lxatac-bsp/recipes-core/systemd/systemd_%.bbappend

```diff
@@ -8,6 +8,8 @@ PACKAGECONFIG:remove = "networkd"
 # is set.
 EXTRA_OEMESON:remove = "-Ddefault-llmnr=no"
 
+WATCHDOG_RUNTIME_SEC = "8"
```

Comment:
Can't this be done via the systemd-conf package? Otherwise changes here will lead to many unnecessary rebuilds. @ejoerns?

Reply:
IMHO, if it's a distro-like decision, I'd find it fine to put it in the Distro (or in this .bbappend, which I find less appropriate).

If that's a per-HW-decision, the original solution of using config drop-ins should be the preferred one.

Reply:
The watchdog value is a hardware-based decision. I've replaced the commit that set `WATCHDOG_RUNTIME_SEC` with one that documents why we are not using it.